gokart/conflict_prevention_lock/task_lock_wrappers.py

```diff
@@ -83,19 +83,18 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def wrap_run_with_lock(run_func: Callable[[], Any], task_lock_params: TaskLockParams):
+def wrap_run_with_lock(run_func: Callable[[], None], task_lock_params: TaskLockParams) -> Callable[[], None]:
     @functools.wraps(run_func)
-    def wrapped():
+    def wrapped() -> None:
         task_lock = set_task_lock(task_lock_params=task_lock_params)
         scheduler = set_lock_scheduler(task_lock=task_lock, task_lock_params=task_lock_params)
 
         try:
             logger.debug(f'Task RUN lock of {task_lock_params.redis_key} locked.')
-            result = run_func()
+            run_func()
             task_lock.release()
             logger.debug(f'Task RUN lock of {task_lock_params.redis_key} released.')
             scheduler.shutdown()
-            return result
```

Comment:
Though I cannot remember, there must be some reason that this method has return, so I think we should reconsider before removing it.